src/ucs/sys/compiler_def.h

```diff
@@ -250,4 +250,15 @@
         *(_b) = __tmp; \
     }
 
+/*
+ * Function name macro
+ */
+
+#if __GNUC__ > 2 || \
```

Comment:
since we already have unprotected `__func__` usage in UCX codebase and gcc 2.95 dates from 2001, I think we can simply replace `__FUNCTION__` occurences by `__func__`.

Reply:
Thanks!